QuizZero.Maven/src/main/java/QuizZero.java

```diff
@@ -0,0 +1,96 @@
+import static java.lang.System.exit;
+
+public class QuizZero {
+
+    public static void main(String[] args) {
+        QuizZero qz = new QuizZero();
+
+        int score = qz.test();
+        exit(score);
+    }
+
+    public int test() {
+        int runningTotal;
+        return 15 - 15;
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [SA: Nonsensical self computation involving a variable (e.g., x & x) (SA_LOCAL_SELF_COMPUTATION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#sa-nonsensical-self-computation-involving-a-variable-e-g-x-x-sa-local-self-computation).
This method performs a nonsensical computation of a local variable with another reference to the same variable (e.g., x&x or x-x). Because of the nature of the computation, this operation doesn't seem to make sense, and may indicate a typo or a logic error. Double check the computation.